programming-extensions/programming-extension-dataspaces/src/main/java/org/objectweb/proactive/extensions/dataspaces/vfs/VFSNodeScratchSpaceImpl.java

```diff
@@ -293,14 +302,27 @@ public synchronized void close() throws IllegalStateException {
             this.fileSystemManager.close();
         }
         logger.debug("Closed node scratch space");
+
+        closed = true;
     }
 
     private FileObject createEmptyDirectoryRelative(final FileObject parent, final String path)
             throws org.apache.commons.vfs2.FileSystemException {
 
         FileObject f = parent.resolveFile(path);
-        f.delete(Selectors.EXCLUDE_SELF);
-        f.createFolder();
+
+        try {
+            f.delete(Selectors.EXCLUDE_SELF);
+        } catch (org.apache.commons.vfs2.FileSystemException e) {
+            logger.warn("Could not delete directory " + f, e);
```

Comment:
No, it should not because exceptions during a copy are rethrown and not catched and logged as it is the case here.